forward/request_sender.go

```diff
@@ -91,8 +91,10 @@ func (s *requestSender) Send() (res []byte, err error) {
 			return s.ScheduleRetry()
 		}
 
+		identity, _ := s.sender.WhoAmI()
```

Comment:
In this case I chose not to handle the error, because this is a log message. The result will be that the identity won't be printed to the log (because it cannot be resolved).

An alternative might be to print "Unknown identity" in the log, but as you point out, that code path is unlikely.